funnel/__init__.py

```diff
@@ -135,7 +135,6 @@
     requires=['funnel'],
     ext_requires=[
         'pygments',
```

Comment:
We're not using Pygments anymore. That also can be removed.

Reply:
Sure. Will take it up as a new PR